File: Java/297. Serialize and Deserialize Binary Tree.java
public class Codec {

    static TreeNode n;
    // Encodes a tree to a single string.
    public String serialize(TreeNode root) {
        n = root;
      return "";
    }

    // Decodes your encoded data to tree.
    public TreeNode deserialize(String data) {
        return n;
    }
}

File: Python/297. Serialize and Deserialize Binary Tree.py
class Codec:

    def serialize(self, root):
        """Encodes a tree to a single string.
        
        :type root: TreeNode
        :rtype: str
        """
        global r
        r = root
        return ''

    def deserialize(self, data):
        """Decodes your encoded data to tree.
        
        :type data: str
        :rtype: TreeNode
        """
        return r
